doc/read-the-docs-site/reference/writing-scripts/compiler-options-table.rst

```diff
@@ -73,6 +73,12 @@
      - Dump Untyped Plutus Core
 
 
+   * - ``inline-constants``
+     - Bool
+     - False
```

Comment:
What does this `False` mean here?

Reply:
The default value perhaps?

Reply:
Shouldn't the default be true or am I confused?

Reply:
> Shouldn't the default be true or am I confused?

Right.. I don't know how we got `False` here. I'll need to take a look.

Reply:
@ana-pantilie Good catch. Fixed.